apple/internal/ios_rules.bzl

```diff
@@ -129,7 +135,7 @@ def _ios_application_impl(ctx):
             bundle_dylibs = True,
             # TODO(kaipi): Revisit if we can add this only for AppStore optimized builds, or at
             # least only for device builds.
-            package_swift_support = True,
+            package_swift_support = package_swift_support,
```

Comment:
instead of having this here in every rule that packages SwiftSupport, can you instead do this inside the swift dylibs partial?